### PR TITLE
Add system information

### PR DESCRIPTION
**Pull request by Meliox**
Ref #45

### CODE REVIEW

File: pve-mod-gui-sensors.sh [hunk 1]
```diff
@@ -680,13 +667,35 @@ Ext.define('PVE.mod.TempHelper', {\n\
...
 	},
 		}" "$pvemanagerlibjs"
 		fi
 
+		if [[ $enableSystemInfo == "true" ]]; then
+			sed -i "/^Ext.define('PVE.node.StatusView',/ {
+				:a;
+				/title:/!{N;ba;}
+				:b;
+				/'separatorBox.*},/!{N;bb;}
```
Comment (Meliox) [inline pve-mod-gui-sensors.sh:683]: Currently dependent on separator box. Should be before the block:





	{
	    colspan: 2,
	    title: gettext('Kernel Version'),
	    printBar: false,
	    // TODO: remove with next major and only use newish current-kernel textfield
	    multiField: true,
	    //textField: 'current-kernel',
	    renderer: ({ data }) => {
		if (!data['current-kernel']) {
		    return data.kversion;
		}
		let kernel = data['current-kernel'];
		let buildDate = kernel.version.match(/\((.+)\)\s*$/)?.[1] ?? 'unknown';
		return `${kernel.sysname} ${kernel.release} (${buildDate})`;
	    },
	    value: '',
	},

**Comment by Meliox** [comment]
@eremem: Should be working now for testing

**Comment by eremem** [comment]
@Meliox I wrote my feedback in https://github.com/Meliox/PVE-mods/issues/45#issuecomment-2150957113 😕

**Comment by eremem** [comment]
I noticed that the base script version you extended in this PR is not the latest one from the main branch. Is there any particular reason for not extending the latest state? ;)

**Comment by eremem** [comment]
And one more thing:
```
                *)
                        warn "Invalid selection. System information will not be displayed."
                        enableSystemInfo=true
                        ;;
```
enableSystemInfo set to true is what the default value in the prompt says but the warning for the default case is... confusing 😄

**Comment by Meliox** [comment]
> I noticed that the base script version you extended in this PR is not the latest one from the main branch. Is there any particular reason for not extending the latest state? ;)

Well... Still trying to get used to Visual Code. Not working so well... Created a new PR to make it look more clean and easier to review changes.

**Pull request closed without merging**